cobalt/build/configs/BUILD.gn

```diff
@@ -21,6 +21,13 @@ config("cobalt_config") {
   ]
   if (current_toolchain == starboard_toolchain) {
     configs += [ "//${starboard_path}/platform_configuration" ]
+  } else if (current_toolchain == cobalt_toolchain) {
+    assert(
```

Comment:
Instead of doing this assert here, there should be an assert in the BUILDCONFIG.gn that checks `_default_toolchain != host_toolchain` when `is_cobalt`

Reply:
done